Src/FluentAssertions/Specialized/ActionAssertions.cs

```diff
@@ -1,210 +1,27 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Diagnostics;
-using System.Linq;
-using System.Reflection;
-using System.Runtime.CompilerServices;
-using System.Threading.Tasks;
-using FluentAssertions.Common;
-using FluentAssertions.Execution;
-using FluentAssertions.Primitives;
 
 namespace FluentAssertions.Specialized
 {
     /// <summary>
     /// Contains a number of methods to assert that an <see cref="Action"/> yields the expected result.
     /// </summary>
     [DebuggerNonUserCode]
-    public class ActionAssertions : ReferenceTypeAssertions<Action, ActionAssertions>
+    public class ActionAssertions : DelegateAssertions<Action>
     {
-        private readonly IExtractExceptions extractor;
-
-        public ActionAssertions(Action subject, IExtractExceptions extractor)
+        public ActionAssertions(Action subject, IExtractExceptions extractor) : base(subject, extractor)
         {
-            this.extractor = extractor;
             Subject = subject;
```

Comment:
🙃 Redundant assignment?

Reply:
Not redundant, as this initialises the new property which hides the base definition on line 20. 
This means we have a value which is used for populating both an Action and Delegate property.